packit_service/worker/helpers/testing_farm.py

```diff
@@ -1459,6 +1459,49 @@ def _payload_installability(self, distro: str, compose: str) -> dict:
             },
         }
 
+    @implements_fedora_ci_test(
+        "custom",
+        skipif=lambda service_config, project, metadata: not project.get_file_content(
+            path=".fmf/version", ref=metadata.commit_sha
+        ),
```

Comment:
Won't work, this raises

https://github.com/packit/ogr/blob/f68be67a30293c39970fc219feca1323dc831bd8/ogr/services/pagure/project.py#L474-L475

Reply:
Oh, I must be blind, I thought I "copied" this:

https://github.com/packit/packit-service/blob/6e158b33e6dfe5c3a5c8bf2d8c57ea34d3a503c8/packit_service/worker/helpers/testing_farm.py#L658-L665